src/extensions/replay/sessionrecording.ts

```diff
@@ -655,6 +665,7 @@ export class SessionRecording {
                 })
                 // proactively flush the buffer in case the session is idle for a long time
                 this._flushBuffer()
+                this.isIdle = true
```

Comment:
move this check to after flushing the buffer since we don't want to drop any of the things in the buffer because we're idle 🙈